app/Console/Commands/InstallApp.php

```diff
@@ -360,7 +360,7 @@ private function generateKey()
      */
     private function createAdminUser($name, $email, $password)
     {
-        $process = $this->artisanProcess('deployer:create-user', [$name, $email, $password, '--no-email']);
+        $process = $this->artisanProcess('deployer:create-user', [$name, $email, $password, '--no-email --admin']);
```

Comment:
Can you split the `--admin` out into a separate array item

Reply:
Done